### DISCUSSION TITLE
Wizard Component : Examples Default display incorrectly

### DISCUSSION THREAD

**Discussion by jfarissi** [microsoft/fluentui-blazor#2581] [General]
[Default Example](https://www.fluentui-blazor.net/Wizard)

My Page:
![wizard](https://github.com/user-attachments/assets/dca87557-052c-4a3e-bfa0-3ed79d0046c4)

Any Idea ?

**Comment by dvoituron** [comment]
A conflict in the CSS styles (same CSS names)?

**Reply by jfarissi** [reply]
the problem is this link

`<link href="{PROJECT_NAME}.styles.css" rel="stylesheet" />`

**Reply by vnbaaij** [reply]
You need to use your actual project name instead of `{PROJECT_NAME}`

**Reply by jfarissi** [reply]
Exactly 
thanks
